concepticondata/conceptlists/Zalizniak-2020-2591.tsv

```diff
@@ -0,0 +1,2592 @@
+Zalizniak-2020-2591-2427	2427	fibula (anat.)					0	0
+Zalizniak-2020-2591-2428	2428	fever	FEVER	1534			0	0
+Zalizniak-2020-2591-2429	2429	fertile (soil)					0	0
+Zalizniak-2020-2591-2430	2430	fatless					0	0
+Zalizniak-2020-2591-2431	2431	fantastic monster					0	0
+Zalizniak-2020-2591-2432	2432	false, wrong					0	0
+Zalizniak-2020-2591-2433	2433	FALSE	FALSE	3069			0	0
+Zalizniak-2020-2591-2434	2434	eyebright (Euphrasia officinalis)					0	0
+Zalizniak-2020-2591-2435	2435	eye of typhoon					0	0
+Zalizniak-2020-2591-2436	2436	extortioner					0	0
+Zalizniak-2020-2591-2437	2437	evil eye					0	0
+Zalizniak-2020-2591-2438	2438	european kingfisher (Alcedo atthis)					0	0
+Zalizniak-2020-2591-2439	2439	ethnic group					0	0
+Zalizniak-2020-2591-2440	2440	essence					0	0
+Zalizniak-2020-2591-2441	2441	ernest, grave	GRAVE	1773			0	0
+Zalizniak-2020-2591-2442	2442	epidemic					0	0
+Zalizniak-2020-2591-2443	2443	environment					0	0
+Zalizniak-2020-2591-2444	2444	entire	WHOLE	2277			0	0
+Zalizniak-2020-2591-2445	2445	enough	ENOUGH	1881			0	0
+Zalizniak-2020-2591-2446	2446	ending (in grammar)					0	0
+Zalizniak-2020-2591-2447	2447	emotionless					0	0
+Zalizniak-2020-2591-2448	2448	eminent man					0	0
+Zalizniak-2020-2591-2449	2449	electric lamp					0	0
+Zalizniak-2020-2591-2450	2450	effeminate					0	0
+Zalizniak-2020-2591-2451	2451	eclipse	ECLIPSE	2285			0	0
+Zalizniak-2020-2591-2452	2452	drunkard					0	0
+Zalizniak-2020-2591-2453	2453	draught animal					0	0
+Zalizniak-2020-2591-2454	2454	dough	DOUGH	273			0	0
+Zalizniak-2020-2591-2455	2455	domestic (animal)					0	0
+Zalizniak-2020-2591-2456	2456	document (n.)					0	0
+Zalizniak-2020-2591-2457	2457	disgusting	DISGUSTING	2507			0	0
+Zalizniak-2020-2591-2458	2458	disgrace					0	0
+Zalizniak-2020-2591-2459	2459	diamonds (in cards)					0	0
+Zalizniak-2020-2591-2460	2460	detailed, thorough					0	0
+Zalizniak-2020-2591-2461	2461	destined to produce					0	0
+Zalizniak-2020-2591-2462	2462	deprived of, lacking					0	0
+Zalizniak-2020-2591-2463	2463	delicate, subtle (person, question etc.)					0	0
+Zalizniak-2020-2591-2464	2464	death	DEATH	910			0	0
+Zalizniak-2020-2591-2465	2465	daughter-in-law	DAUGHTER-IN-LAW	1057			0	0
+Zalizniak-2020-2591-2466	2466	danger	DANGER	1810			0	0
+Zalizniak-2020-2591-2467	2467	dandelion (Taraxacum)	DANDELION	1800			0	0
+Zalizniak-2020-2591-2468	2468	cultivated					0	0
+Zalizniak-2020-2591-2469	2469	crucian					0	0
+Zalizniak-2020-2591-2470	2470	crowd (of people)	CROWD	2023			0	0
+Zalizniak-2020-2591-2471	2471	crossbill (Loxia)					0	0
+Zalizniak-2020-2591-2472	2472	creditor					0	0
+Zalizniak-2020-2591-2473	2473	cranberry (Oxycoccus)					0	0
+Zalizniak-2020-2591-2474	2474	cornflower (Centaurea)	CORNFLOWER	1960			0	0
+Zalizniak-2020-2591-2475	2475	cornel					0	0
+Zalizniak-2020-2591-2476	2476	core (of plant)					0	0
+Zalizniak-2020-2591-2477	2477	copper	COPPER	1825			0	0
+Zalizniak-2020-2591-2478	2478	contract					0	0
+Zalizniak-2020-2591-2479	2479	continent	MAINLAND	166			0	0
```

Comment:
Not 100% sure whether MAINLAND fits.

Reply:
I would unmap it, too. I think the distinction is more along the lines of Jena being on mainland Germany as opposed to Helgoland.